utils/content.ts

```diff
@@ -161,24 +193,24 @@ export const getContentList = (contentType: IContentType) => {
       throw new Error('You have to provide a content type');
   }
 
-  const content: { date?: Date; previewImage: any; id: string }[] = contentFiles
+  const content = contentFiles
     .filter((contentFile) => contentFile.endsWith('.md'))
-    .map((contentFile) => {
-      const filePath = `${contentDir}/${contentFile}`;
-      const rawContent = fs.readFileSync(filePath, {
+    .map((contentItem) => {
+      const contentPath = `${contentDir}/${contentItem}`;
+      const rawContent = fs.readFileSync(contentPath, {
```

Comment:
changed the variable names since they were already declared in line 2 and line 196